docs/commandline.rst

```diff
@@ -139,3 +139,19 @@ For example:
 ::
 
     kinto rebuild-quotas --ini=config/postgresql.ini
+
+Flush Cache
+-----------
+
+Clears the Memory Backend Cache This can be useful for
+debugging. 
+
+::
+
+    usage: Kinto flush-cache [--ini INI_FILE]
```

Comment:
I would remove `usage:`, and also write `Kinto` with lowercase

Reply:
@leplatrem Done, although can you elaborate as to why the `usage: `should was to be removed? I see that every other function description in the rst file has it.

Reply:
I think I had a pending review that was never submitted and it got sent when I sent some comments yesterday.
I'm really sorry for the confusion!

Reply:
Okay, you are right! the `usage: `  is everywhere! Keep it!